x/evm/module.go

```diff
@@ -168,12 +167,11 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 
 // RandomizedParams creates randomized evm param changes for the simulator.
 func (AppModule) RandomizedParams(r *rand.Rand) []simtypes.ParamChange {
```

Comment:
nit, it's a good practice to ignore the args if they are not being used

```suggestion
func (AppModule) RandomizedParams(_ *rand.Rand) []simtypes.ParamChange {
```

Reply:
you can also update the comments to reflect the new changes (i.e no-ops)